client/layout/universal-navbar-header/nav-style.scss

```diff
@@ -28,6 +28,21 @@ $studio-blue-30: #0675c4;
 	}
 }
 
+.is-section-themes,
+.is-section-theme {
+	&.has-no-sidebar {
+		.masterbar {
+			height: 56px;
+			position: relative;
```

Comment:
Not related to the PR, but I noticed a missing bottom border on the sticky search bar:

![Screenshot 2022-11-03 at 12 24 18](https://user-images.githubusercontent.com/8436925/199697447-c1166846-4978-4cd3-9186-88deb6e82d64.png)

Is this a bottle we can pick while we're here?